src/renderer/api/api-manager.ts

```diff
@@ -23,6 +23,12 @@ export class ApiManager {
 
   registerApi(apiBase: string, api: KubeApi) {
     if (!this.apis.has(apiBase)) {
+      this.stores.forEach((store) => {
+        if(store.api === api) {
+          this.stores.set(apiBase, store);
+        }
+      });
+
```

Comment:
```suggestion
      const store = this.getStore(api);
      if (store) this.stores.set(apiBase, store);
```

Reply:
This does not work since api has updated value for `apiBase` and no stores are found with that key.

Reply:
Ok, but sounds like a bug since `apiManager.getStore(api: string | KubeApi)`..
Introduced from here: https://github.com/lensapp/lens/pull/2091